src/constructor.io/models/common/VariationsMap.cs

```diff
@@ -10,6 +10,8 @@ public class Value
         public string Aggregation { get; set; }
         [JsonProperty("field")]
         public string Field { get; set; }
+        [JsonProperty("value")]
+        public string CountedValue { get; set; }
```

Comment:
I think we should keep to how the API names fields in case `value` is used in another aggregation type in the future. wdyt?

```suggestion
        public string Value { get; set; }
```

Reply:
The data type change makes sense.

In regards to the name, .NET has a limitation where the property can't have the same name as the class it's within, and changing the class name at this point would be a bigger change than planned